pkg/config/polling_manager.go

```diff
@@ -61,13 +62,11 @@ type PollingProjectConfigManager struct {
 // OptionFunc is a type to a proper func
 type OptionFunc func(*PollingProjectConfigManager)
 
-// DefaultRequester is an optional function, sets default requester based on a key.
-func DefaultRequester(sdkKey string) OptionFunc {
+// DefaultRequester is an optional function, sets default requester
+func DefaultRequester() OptionFunc {
```

Comment:
I am not sure we need this OptionFunc as it is never used. We always instantiate a HTTPRequester in the `NewPollingProjectConfigManager` method.

Reply:
removed